gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -383,26 +393,38 @@ def get_combine_faf_resources(
     # Create resource collection for each step of the pipeline.
     combined_frequency = PipelineStepResourceCollection(
         "--create-combined-frequency-table",
-        output_resources={"comb_freq_ht": get_combined_frequency(test=test)},
+        output_resources={
+            "comb_freq_ht": get_combined_frequency(
+                test=test, filtered=apply_release_filters
+            )
+        },
         input_resources={
             "generate_freq.py": {"exomes_ht": get_freq(test=test)},
             "generate_freq_genomes.py": {
                 "genomes_ht": get_freq(test=test, data_type="genomes")
             },
+            "final_filter.py": {"exomes_filter_ht": final_filter(data_type="exomes")},
+            "final_filter_genomes.py": {
```

Comment:
since the filters HTs are getting added to `input_resources`, the default behavior should be to apply filters (rather than needing to set an argument to apply filters), right?

Reply:
Same comment above to prevent a rerun I would like to keep as is, but I can change this to only add them if `apply_release_filters` is true